examples/glTFViewer/glTFViewer.cpp

```diff
@@ -252,10 +254,24 @@ namespace {
             .SetBindingsType(nxt::ShaderStageBit::Fragment, nxt::BindingType::SampledTexture, 2, 1)
             .GetResult();
 
+        auto renderpass = device.CreateRenderPassBuilder()
+            .SetAttachmentCount(1)
+            .AttachmentSetFormat(0, nxt::TextureFormat::R8G8B8A8Unorm)
+            .SetSubpassCount(1)
+            .SubpassSetColorAttachment(0, 0, 0)
+            .GetResult();
+        auto framebuffer = device.CreateFramebufferBuilder()
+            .SetRenderPass(renderpass)
+            // attachment 0 -> back buffer
+            // (implicit) // TODO(kainino@chromium.org): use the texture provided by WSI
+            .SetDimensions(640, 480)
```

Comment:
It's weird to me that screen dimensions are now part of the material info. But I think it's correct?